### ISSUE TITLE
Icon embedded into executable [Windows]

### ISSUE THREAD

**Issue by smaragdus** [MasterQ32/kristall#138]
I discovered Gemini protocol just recently and [Kristall](https://kristall.random-projects.net/) is the second Gemini client I am testing. I find [Kristall](https://github.com/MasterQ32/kristall) rather promising as it is cross-platform and supports other protocols beside Gemini (Gopher support is essential for me as there does not seem to be many graphical Gopher clients). I may suggest several enhancements and the first one will be rather cosmetic one.

I tested Windows nightly build (`kristall-nightly.zip`) (presumably version [0.3](https://github.com/MasterQ32/kristall/releases/tag/V0.3)) and I noticed that the program icon, which appears in the title bar and in about window, screen:

![Kristall 0 3 - 2021-01-31 - 005](https://user-images.githubusercontent.com/9025757/106371186-0cf50700-636a-11eb-9718-7959a01dd05e.png)

is not embedded into the executable (`kristall.exe`), screen:

![Kristall 0 3 - 2021-01-31 - 009](https://user-images.githubusercontent.com/9025757/106371429-e2f11400-636c-11eb-9609-efa65c140529.png)

So I would like to suggest the program icon to be embedded into the executable (`kristall.exe`) and thus the users would not need to download [icons](https://github.com/MasterQ32/kristall/tree/master/src/icons) to create nicer short cuts for the program. I suppose that this request would be very easy to be implemented.

**Comment by mikejzx** [comment]
Thanks for the well-written issue.
This should be fixed as of #140 
Please try running the latest nightly build from [here (near bottom of page)](https://kristall.random-projects.net/) to confirm that the issue is fixed 😉

**Comment by smaragdus** [comment]
@mikejzx 

Hello and thanks for the notification.

I downloaded the [Windows nightly build](https://kristall.random-projects.net/download/kristall-nightly.zip) but it is the same version which I have already tested (from _2020-06-30_), hashes:

```
Name: kristall-nightly.zip
File Size: 28112588 Byte(s) (26.81 MB)
Modified Date: 2020-06-30 20:05
MD5: 0abfe3d2832386c7a3ea33cacacf1270
SHA1: 1d182264b08130a29e223331d3d9c980e4c4ab99
SHA256: 2199d41ec8408d1bcb61056764d826d95f4b2c19c7d1a5ce7d1d9bec009b7252
SHA512: 37750df6f012ceb6c9f28af3fbf87051c736423e3aa1dffc1956cae5d4d388e663e864d49d09934f6d7e46bb7574863cb6c83bd73b86e6d9d3d309827994d15c
RIPEMD160: 481dd3ad09301cbc492ff45d3dddbfdb17fb5cab
```
I used this download link:

`https://kristall.random-projects.net/download/kristall-nightly.zip`

If there is a new nightly version compiled for Windows (as indicated in the [Change Log](https://github.com/MasterQ32/kristall/blob/master/src/about/updates.gemini)) it seems that it hasn't already been updated on [Kristall](https://kristall.random-projects.net/) website. If new Windows nightly build is published I will download it and test it.

Regards

**Comment by mikejzx** [comment]
@smaragdus 

Sorry. The nightly build system was broken at the time you tested it. It was fixed just recently so you try the download link again

[https://kristall.random-projects.net/download/Kristall-nightly-windows-x86_64.zip](https://kristall.random-projects.net/download/Kristall-nightly-windows-x86_64.zip)

**Comment by smaragdus** [comment]
@mikejzx 

Thanks for the new notification. I downloaded [the latest nightly](https://kristall.random-projects.net/download/Kristall-nightly-windows-x86_64.zip) (from _2021-02-26_, supposedly version **0.4**) and I can confirm that the executable (`kristall.exe`) has been iconized, screen:

![Kristall 0 4 - 2021-02-27 - 003](https://user-images.githubusercontent.com/9025757/109396846-30e14380-793c-11eb-975c-741c2b19feb4.png)

Thank you for the implementation of the tiny enhancement that I suggested. Now creating shortcuts for [Kristall](https://github.com/MasterQ32/kristall) would be easier and handier. 

I will continue testing [Kristall](https://kristall.random-projects.net/) and I may suggest more enhancements in the future.

I will close this issue as it is resolved.

Regards
